src/lib.rs

```diff
@@ -55,6 +55,7 @@ pub mod scale;
 pub mod utils;
 
 use drawille::Canvas as BrailleCanvas;
+pub use drawille::PixelColor;
```

Comment:
Could we add our own enum for colors into `lib.rs`? I don't want implementation details to be seen in the public interface

Reply:
Thoughts on instead using a more widely used crate to represent color in an attempt to improve interop within projects? For example, https://crates.io/crates/rgb

Reply:
Rgb looks fine 👍

Reply:
Switched to the rgb crate for handling color